sdk/eidos_sdk/memory/agent_memory.py

```diff
@@ -281,21 +278,45 @@ async def query(
             )
         return returnDocuments
 
+    async def raw_query(
```

Comment:
What is this needed for? it looks like the only user grabs the embeddings the same way the normal query does. If it is the query item vs document wrapper, perhaps we should soothe over those differences instead.